.github/workflows/example-project.yml

```
@@ -80,3 +79,23 @@ jobs:
         working-directory: example-project/usage-from-c
         run: |
           make
+
+      - name: Install MSYS2
+        if: startsWith(matrix.os, 'windows') && matrix.toolchain-suffix == '-gnu'
+        uses: msys2/setup-msys2@v2
+        with:
+          release: false
```

```suggestion
        with:
          release: false
          install: >-
            mingw-w64-x86_64-pkgconf
            mingw-w64-x86_64-gcc
            base-devel
            git

```
Maybe these packages are needed?